src/test/scala/com/fulcrumgenomics/illumina/SampleTest.scala

```diff
@@ -41,13 +41,16 @@ class SampleTest extends UnitSpec with OptionValues{
   }
 
   it should "support extended attributes" in {
-    val sample = new Sample(0, "ID", "NAME", "LIBRARY", extendedAttributes=Seq(("FOO", "1"), ("BAR", "2  ")).toMap)
+    val sample = new Sample(0, "ID", "NAME", "LIBRARY", extendedAttributes=Seq(("FOO", "1"), ("BAR", " 2  "), ("BAZ", "  ")).toMap)
```

Comment:
how about an attribute with only spaces?

Reply:
Only spaces for the key, or value? The latter was added `("BAZ", "  ")`.

I think a requirement of the spec. would be to have no key-less values (whitespace only). I could add that requirement here?

https://github.com/fulcrumgenomics/fgbio/blob/3183c233307250564088561fad22a83f368e4cda/src/main/scala/com/fulcrumgenomics/illumina/Sample.scala#L61